input/pagecontent/changes.md

```diff
@@ -11,7 +11,7 @@ This change log documents the significant updates and resolutions implemented fr
 - Changed the Observation  search parameter 'patient' from SHALL to MAY [FHIR-47171](https://jira.hl7.org/browse/FHIR-47171)
 - Removed the requirement for including an offset in the Patient 'birthdate' search parameter [FHIR-47150](https://jira.hl7.org/browse/FHIR-47150)
 - Updated requirement for AU Core Requester to mandate both system and code values for identifier search parameters on Location, Organization, Practitioner and PractitionerRole [FHIR-46782](https://jira.hl7.org/browse/FHIR-46782)
-
+- Changed MedicationRequest search parameters 'patient+intent' and 'patient+intent+status' from SHALL to SHOULD; added new MedicationRequest search parameter 'patient+status' with required support of SHALL [FHIR-47073](https://jira.hl7.org/browse/FHIR-47073)
```

Comment:
Change to something like: "Changed MedicationRequest search parameters 'patient+intent' and 'patient+intent+status' from SHALL to SHOULD, and added a new MedicationRequest search parameter 'patient+status' as a SHALL [FHIR-47073](https://jira.hl7.org/browse/FHIR-47073)"

Reply:
Amended as suggested, thanks.